kinder/ci/tools/update-workflows/config.yaml

```diff
@@ -156,6 +156,18 @@ jobGroups:
   - kubernetesVersion: -2
   - kubernetesVersion: -3
 
+- name: dryrun-regular
```

Comment:
thanks for the PR @SataQiu 
as noted in https://github.com/kubernetes/kubeadm/issues/2653

i think we can call this workflow group just `dryrun` and make it also include `upgrade`